pages/books/Title.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import Typography from '@mui/material/Typography';
+
+interface TitleProps {
```

Comment:
you could use the React `PropsWithChildren` type here

Reply:
done